f8a_worker/storages/postgres.py

```diff
@@ -100,3 +104,57 @@ def get_analysis_by_id(self, analysis_id):
             one()
 
         return found
+
+    def check_api_user_entry(self, email):
+        """Check if a user entry has already been made in api_requests
+
+        :param email: str, user's email id
+        :return: count of user's entry in api_requests
+        """
+        return PostgresBase.session.query(APIRequests).\
+            filter(APIRequests.user_email == email).first()
+
+    def store_in_bucket(self, content):
+        s3 = StoragePool.get_connected_storage('S3UserProfileStore')
+        s3.store_in_bucket(content)
```

Comment:
I have added Api_Requests table to the Bayesian postgresql database and that was reason behind placing the login in `BayesianPostgres` class. Can you please help me understand why a new adapter is appropriate in this case? I must have missed something really obvious.

Moreover, I am quite handicapped ATM with the concept of database adapters.